test/smoke/appServers/global-resources/telemetryprocessors_applicationinsights.json

```diff
@@ -89,7 +80,7 @@
         },
         {
           "type": "log",
-          "name": {
+          "body": {
             "fromAttributes": [
               "applicationinsights.internal.logger_name","applicationinsights.internal.log_level"
             ],
```

Comment:
it would be nice to instead test the PII data redaction scenario, but we can do that in follow-up PR later